src/taskgraph/main.py

```diff
@@ -582,19 +579,16 @@ def show_taskgraph(options):
 )
 def build_image(args):
     from taskgraph.config import load_graph_config  # noqa: PLC0415
-    from taskgraph.docker import build_context, build_image  # noqa: PLC0415
+    from taskgraph.docker import build_image  # noqa: PLC0415
 
     validate_docker()
+    graph_config = load_graph_config(args["root"])
 
-    root = args["root"]
-    graph_config = load_graph_config(root)
-
-    if args["context_only"] is None:
-        build_image(args["image_name"], args["tag"], graph_config, os.environ)
-    else:
-        build_context(
-            args["image_name"], args["context_only"], graph_config, os.environ
-        )
+    return build_image(
+        graph_config,
+        args["image_name"],
+        args["context_only"],
```

Comment:
The new save_image isn't really the same thing as the old context_only?

Reply:
Unless you meant it as an actual question, in which case, no context_only just saves the docker-context as generated by the decision task, it doesn't actually build the image.

I kind of question the usefulness of this, but it was easy enough to preserve *shrug*

Reply:
My mistake, I misread the arguments to taskgraph.docker.build_image